casadm/cas_lib.c

```diff
@@ -1119,8 +1119,16 @@ int stop_cache(uint16_t cache_id, int flush)
 	int status;
 
 	/* Don't stop instance with mounted filesystem */
-	if (cache_mounts_detected(cache_id))
+	int cmplen = 0;
+	char pattern[80];
+
+	/* verify if any core (or core partition) for this cache is mounted */
+	cmplen = snprintf(pattern, sizeof(pattern), "/dev/cas%d-", cache_id) - 1;
```

Comment:
Again, removing null from the end, which shouldn't be there...?

Reply:
I stand by the version that it should :)